DEV-Simple/Utilities/ThemeManager.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ThemeManager.swift
+//  DEV-Simple
+//
+//  Created by Daniel Chick on 11/27/20.
+//  Copyright © 2020 DEV. All rights reserved.
+//
+
+import UIKit
+import ForemWebView
+
+class ThemeManager {
+    private(set) static var useDarkMode = false
+
+    static func applyTheme(to viewController: ViewController) {
+        guard let webView = viewController.webView, let theme = webView.userData?.theme() else { return }
```

Comment:
Try this:
1. Log in
1. Choose dark or pink theme
1. Refresh from the main feed
1. Log out

The shell will remain stuck on that theme as soon as the user logs out because the `userData` becomes `nil` and the `guard` clause won't update back to the default theme. This small change helps fix that edge case

```suggestion
        let theme = viewController.webView.userData?.theme() ?? .base
```

Reply:
It looks like the hacker theme has a problem even if adding this suggestion. This will also require to update the native button colors based on the theme.

Reply:
Thanks for testing that edge case! I incorporated a fix in 6b9dcff